crt_portal/cts_forms/migrations/0033_auto_20200121_2151.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.8 on 2020-01-21 21:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0032_auto_20200116_2255'),
```

Comment:
Want to rename this file `0034_update_election_choices.py` and change the dependency here to `0033_merge_20200121_1822` that way we won't have to merge the migrations?

Reply:
Yes!! I would love to avoid more merge migrations 😄 .

Reply:
this works for me!!